provision/filter_plugins/aws_sg_rules.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
```

Comment:
Add comments to this file to explain the intent of the file and intent of functions contained within.